### DISCUSSION TITLE
next/image image not loading : getting 503 (Service Unavailable) in simple image example

### DISCUSSION THREAD

**Discussion by stajics** [vercel/next.js#32778] [Help]
### What version of Next.js are you using?

10.2.3

### What version of Node.js are you using?

14.15.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

vercel

### Describe the Bug

Using a new next.js project. Optimized images won't load from a specific domain.  

For 
```
<Image src="https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg" width={500} height={500} />
```
getting 503 (Service Unavailable) error.
<img width="373" alt="Screen Shot 2021-05-31 at 4 46 26 PM" src="https://user-images.githubusercontent.com/17711182/120210642-4c583500-c230-11eb-9f62-fd75c79ac461.png">

Adding `unoptimized` prop shows the image.

### Expected Behavior

Image should load from any domain

### To Reproduce

Create a new next app.

Create `next.config.js` and add 
```
module.exports = {
    images: {
        domains: ['www.jobboard.com']
      },
}
```

Add 
```
<Image src="https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg" width={500} height={500} />
```
to a page and navigate to the page. Image will be broken

**Comment by manowski** [comment]
I had the same issue and was wondering why it doesn't work. As a result, I discovered that some urls containing images are not in the image format. That's why you're getting an error and the image won't load.
Your image, for example, is in html format and you've put it in the image component.

You can check the format here: aconvert.com/analyze.html

**Comment by balazsorban44** [comment]
A simple `curl https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg` shows that you return HTML, instead of an image response.

```js
await fetch(
  "https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg"
)
```

Also gives back 503, so the issue is probably not with `next/image`.

**Reply by dewaleolaoye** [reply]
were you able to resolve this issue?

**Comment by dewaleolaoye** [comment]
Has anyone resolved this issue?

**Comment by theskillwithin** [comment]
any news on this?

**Reply by dewaleolaoye** [reply]
No fix works for me, I had to switch to `img` tag

**Comment by bernardoosthuizen** [comment]
Hi all. Has this been resolved? Ever since upgrading to Next13, next/image has given us a 503. Using <img> seems to resolve the issue but its obviously not ideal.

**Reply by Pseudoman21** [reply]
Still haven't found a solution for this, unfortunately.

**Comment by Mandy9943** [comment]
same issue over here :(

**Reply by sstrubberg** [reply]
Just ran into this myself.

**Comment by fimbres** [comment]
I'm running into this issue deploying Nextjs 13.5 into GCP with Cloud Run. any updates?

**Comment by sfonua10** [comment]
Are you running into the issue in prod only and it works locally? If so, it's probably the execution time limit. It's set to 10 seconds for the Hobby plan (free tier). If it's longer than that to upload the image, you could be getting 503 errors. Same with memory limit.

**Comment by iWolf22** [comment]
Still getting this issue (error 503) running NextJS `^14.2.4` locally and in production (dockerized environments) with both "remote" and "local" images. I have added Sharp `^0.33.4` as per the NextJS documentation and followed the remote image guidelines once again as per [Remote Images NextJS Docs](https://nextjs.org/docs/pages/building-your-application/optimizing/images#remote-images), see below.

```javascript
/** @type {import('next').NextConfig} */
const nextConfig = {
images: {
        remotePatterns: [
            {
                hostname: "flagcdn.com",
                pathname: "/w40/**",
                port: "",
                protocol: "https",
            },
        ],
    },
};
export default nextConfig;
```

I have also tried messing with the [Loaders](https://nextjs.org/docs/app/building-your-application/optimizing/images#loaders) and the [Priority](https://nextjs.org/docs/app/building-your-application/optimizing/images#priority) props but to no avail. From my research it appears the NextJS images seem to only work consistently on Vercel, it would be nice if they consistently worked elsewhere too (seems to be a very common problem, for now I will default to a generic `<img />` tag while waiting for some help)!

**Images Randomly Failing to Load**
![image](https://github.com/user-attachments/assets/fa4d6384-fee8-47e2-a426-79e7e2971ed4)

**Console Log Errors**
![image](https://github.com/user-attachments/assets/3dedf081-8d6b-4dac-9091-b5e205e4397e)

**Related Conversations**
 - https://github.com/vercel/next.js/issues/31610
 - https://github.com/serverless-nextjs/serverless-next.js/issues/840
 - https://stackoverflow.com/questions/71392729/next-image-component-url-issue
 - https://stackoverflow.com/questions/70457750/nextjs-not-fetching-images-using-image-url-after-deploying-on-production-server
 - https://stackoverflow.com/questions/73919564/getting-503-error-on-some-image-using-next-image-and-aws-s3-with-hostend-on-ampl
 - https://stackoverflow.com/questions/66637391/next-images-components-are-too-slow-to-appear

**Comment by iWolf22** [comment]
I am not sure if this fixes the `error 503` problem 100% of the time, but a second work around I found at [this stack overflow thread](https://stackoverflow.com/questions/70457750/nextjs-not-fetching-images-using-image-url-after-deploying-on-production-server/71377671#71377671) claims that providing your own loader can help fix the issue.

```diff
<Image
  alt={option.name + " flag"}
  height={20}
  key={option.code}
+  loader={({ src, width }: ImageLoaderProps) => {
+     return `https://flagcdn.com/w40/${src}?w=${width}`;
+  }}
  src={`${option.code.toLowerCase()}.png`}
  width={40}
/>
```

Not too sure why this is the case but I assume it has something to do with "A loader is a function that generates the URLs for your image" and as stated in the [Loaders](https://nextjs.org/docs/app/building-your-application/optimizing/images#loaders) NextJS docs.

**Comment by amerhamdan3** [comment]
Has anyone resolved this issue? i have the same issue with google run

**Comment by ShayokhShorfuddin** [comment]
I am using Nextjs 15. I have hosted my application using MellowHost Cpanel. Some of my images take too long to load and eventually the servers returns 503 error.

![image](https://github.com/user-attachments/assets/51414359-c7b2-4e62-ab77-38c36247b172)
Any solution?

**Comment by sundaram123krishnan** [comment]
any solution?
i have hosted my app in an vps and using uploadthing,
